census_view.hpp

```diff
@@ -38,16 +38,16 @@
 #include <vector>
 
 class CensusDocument;
-class CensusViewDataViewModel;
+class CensusViewTable;
```

Comment:
This seems to show that it would be more consistent to call this `CensusViewGridTable`.